src/axes.jl

```diff
@@ -431,9 +433,18 @@ function expand_extrema!(sp::Subplot, plotattributes::AKW)
         data = plotattributes[if vert
             letter
         else
-            letter == :x ? :y : letter == :y ? :x : :z
```

Comment:
I'd rather have the ternary operator here

Reply:
odd, let me see what I can do on this.

Reply:
ok, that is because `BlueStyle` does not allow chaining, I've reverted it to using `DefaultStyle`.